docs/cognito.md

```diff
@@ -0,0 +1,10 @@
+# AWS Cognito OAuth
+
+A wrapper around the [Passport Cognito OAuth](https://www.npmjs.com/package/passport-cogito-oauth2) package.
+
+### Configuration
+
+- `COGNITO_CONSUMER_CLIENT` _(required)_: the client key setup in AWS cognito.
```

Comment:
should this be `COGNITO_CONSUMER_KEY` to be consistent with other provider env vars?

Reply:
I actually had it that way at the beginning sign that is how it was set with google.  But I changed it since it maps to the clientId, I thought it made more sense as client.  Twitter is the only one that uses the term CONSUMER.  But I am happy to change it to consumer to be consistent.

Reply:
Updated to use KEY to be consistent.